<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_lap_ket_qua_cham_thi</class>
 <widget class="QWidget" name="man_hinh_lap_ket_qua_cham_thi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>581</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:None;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="ten_giang_vien">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>25</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 255);
border: 2px solid rgba(46, 82, 101, 200);
border-radius: 10px;
padding: 0 8px;
font-size: 16px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource.qrc">:/images/Ảnh26.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>55</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource.qrc">:/images/sinh viên.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: None;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Tên giảng viên:</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>ten_giang_vien</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>231</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgba(46, 82, 101, 200);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0, 112, 192);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,3">
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="lop_hoc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Lớp học: </string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="mon_hoc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Môn học:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="ma_de">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Mã đề:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="them">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 170, 255);
border-radius: 10px;border: 2px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>Thêm</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource.qrc">
         <normaloff>:/images/6_n.png</normaloff>
         <normalon>../../../cnpm/6_n.png</normalon>:/images/6_n.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="thoat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);
border-radius: 10px;border: 2px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>Thoát</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource.qrc">
         <normaloff>:/images/32_n.png</normaloff>
         <normalon>../../../cnpm/32_n.png</normalon>:/images/32_n.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="tim_kiem">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 170, 255);
border-radius: 10px;
border: 2px solid rgb(0,0,0)</string>
       </property>
       <property name="text">
        <string>Tìm kiếm</string>
       </property>
       <property name="icon">
        <iconset>
         <disabledoff>:/images/893_n.png</disabledoff>
        </iconset>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;DANH SÁCH NHẬP ĐIỂM THI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ket_qua_cham_thi">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>691</width>
     <height>381</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgba(46, 82, 101, 200);
background-color: rgb(255, 255, 255);
border-radius: 0px;
border-color: rgb(0, 112, 192);</string>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
